Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -84,6 +84,22 @@ void JitArm64::Break(UGeckoInstruction inst)
 	exit(0);
 }
 
+bool JitArm64::MergeAllowedNextInstructions(int count)
```

Comment:
Can we share the implementation of this function with Jit64, rather than copy-pasting it?